src/markdown/CustomMarkdownEmitter.ts

```diff
@@ -155,13 +155,16 @@ export class CustomMarkdownEmitter extends MarkdownEmitter {
                 }
             }
         } else if (result.errorMessage) {
+            const elementText = docLinkTag.codeDestination!.emitAsTsdoc();
```

Comment:
Can it lead to possible exception? I have not checked what this function does, just a sanity check.

Reply:
It's a fair question. There is another location above that does the same non-null assertion, so I figured it was safe to do the same here. It is probably better to check for it above and throw an explicit error though. I'll do that.